README.md

```diff
@@ -1 +1,68 @@
-# data-feed-proxy-combinators
+# @api3/data-feed-proxy-combinators
+
+> For general information about API3 data feeds, dAPIs, and other services, end users are recommended to refer to the [official API3 dApp documentation](https://docs.api3.org/dapps/).
+
+## What is this?
+
+This repository contains a suite of utility smart contracts designed to work with and extend the functionality of API3 data feeds. These contracts, often referred to as "proxy combinators" or "adapters," act as wrappers around existing data feeds, allowing dApp developers to transform, derive, or normalize data feed values to suit their specific application needs.
+
+The content in this directory provides insights into these combinator contracts, their design, and their intended use cases. For low-level documentation about functionality and usage, refer to the Natspec docstrings and the implementation of the respective contracts.
+
+## Overview
+
+API3 provides reliable, first-party oracle data directly to smart contracts, primarily through dAPIs (decentralized APIs) which are continuously updated data feeds. While these dAPIs serve a wide array of use cases, dApps occasionally require data in a format or derivation not directly offered by a base dAPI. For instance, a dApp might need an inverted price (e.g., USD/ETH from an ETH/USD dAPI), a value scaled to different decimal precision, or a data point derived from combining multiple existing dAPIs (e.g., BTC/ETH from a BTC/USD dAPI and an ETH/USD dAPI).
+
+Manually implementing such logic within each dApp can be repetitive, error-prone, and gas-intensive. Furthermore, requiring a new, dedicated first-party oracle deployment for every minor data variation is not scalable. Data Feed Proxy Combinators address these challenges by providing **standardized and reusable smart contract solutions** for adapting data feeds on-chain. While a specific instance of a combinator proxy is typically deployed to serve a particular dApp's needs (often in conjunction with its dedicated underlying data feed proxy), these combinator contracts offer a common, audited approach to performing such transformations, promoting code reuse and reliability across the ecosystem.
+
+### Core Capabilities
+
+These combinator contracts are designed to be modular and composable, typically taking the address(es) of underlying feed(s) and any necessary operational parameters (like scaling factors) in their constructors. They offer several key capabilities:
+
+- **Transformation:** Modify the value or metadata of a single underlying data feed.
```

Comment:
Imo just list the contracts and their use. There is no need to categorize them into "transformation/derivation/etc..."